src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -1018,7 +1018,7 @@ Namespace Microsoft.CodeAnalysis.Semantics
 
         Private Function CreateBoundCatchBlockOperation(boundCatchBlock As BoundCatchBlock) As ICatchClause
             Dim handler As Lazy(Of IBlockStatement) = New Lazy(Of IBlockStatement)(Function() DirectCast(Create(boundCatchBlock.Body), IBlockStatement))
-            Dim caughtType As ITypeSymbol = Nothing ' Manual
+            Dim caughtType As ITypeSymbol = boundCatchBlock.ExceptionSourceOpt?.Type
```

Comment:
>im caughtType As ITypeSymbol = boundCatchBlock.ExceptionSourceOpt?.Type [](start = 13, length = 71)

I do not see a single test covering this change. If that is correct, please add one.

Reply:
done